components/annot-missing-values.vue

```diff
@@ -68,133 +65,41 @@
 
             ...mapGetters([
 
-                "isMissingValue",
                 "valueDescription"
-            ])
-        },
-
-        created() {
-
-            // 1. Create lists of potentially missing values for the data based on
-            // information in the data dictionary and the data type for this category
-            this.determineMissingValues();
-
-            // 2. Create an array of objects for missing value table data
-            this.createTableData();
-        },
-
-        methods: {
-
-            createTableData() {
-
-                // 0. Wipe any old table data
-                this.tableItems = [];
-
-                // 1. Create a list of objects describing each value for each
-                // column listed in 'uniqueValues'
-                for ( const columnName in this.uniqueValues ) {
-                    for ( const value of this.uniqueValues[columnName] ) {
-
-                        // A. Only missing values for this data type are listed
-                        if ( this.isMissingValue(columnName, value) ) {
-
-                            this.tableItems.push({
-
-                                column: columnName,
-                                description: this.valueDescription(columnName, value),
-                                value: value
-                            });
-                        }
-                    }
-                }
-            },
-
-            determineMissingValues() {
-
-                // 0. Create an object for missing values lists for all columns of this component
-                const missingValuesLists = {};
-
-                // 1. Create a set of lists for potentially missing values of
-                // each column assigned to this tab's cateogry
-                for ( const columnName in this.uniqueValues ) {
-
-                    // A. Only determine missing values if there is no value list for this column in the store
-                    if ( Object.keys(this.missingColumnValues).includes(columnName) ) {
-                        continue;
-                    }
+            ]),
 
-                    // B. Each column has a list of potentially missing values
-                    missingValuesLists[columnName] = [];
+            ...mapState([
 
-                    // C. Determine value invalidity based on the data type of this category
-                    for ( const value of this.uniqueValues[columnName] ) {
+                "missingColumnValues"
+            ]),
 
-                        switch ( this.dataType ) {
+            tableItems() {
```

Comment:
Remember to add semicolons. A couple places in this function. Not sure why our linter is not picking up on this.

Reply:
I checked, it's not set as a rule. Try `npx eslint --print-config components/annot-missing-values.vue`. In fact, `no-extra-semi` is set in the rules. Feel free to add to https://github.com/neurobagel/annotation_tool/issues/125 if you find other things